core/src/test/java/org/apache/iceberg/TableTestBase.java

```diff
@@ -307,6 +353,30 @@ ManifestFile writeDeleteManifest(int newFormatVersion, Long snapshotId, DeleteFi
     return writer.toManifestFile();
   }
 
+  InputFile writeManifestList(String compressionCodec, ManifestFile... manifestFiles)
+      throws IOException {
+    File manifestListFile = temp.newFile();
+    Assert.assertTrue(manifestListFile.delete());
+    OutputFile outputFile =
+        org.apache.iceberg.Files.localOutput(
+            FileFormat.AVRO.addExtension(manifestListFile.toString()));
+
+    try (FileAppender<ManifestFile> writer =
+        ManifestLists.write(
+            formatVersion,
+            outputFile,
+            SNAPSHOT_ID,
+            SNAPSHOT_ID - 1,
+            formatVersion > 1 ? 34L : 0,
```

Comment:
I'm guessing we are just writing some random number here to not cause conflicts?

Reply:
This was written by Sumeet and I don't know why he used 34L, but I guess so.

Reply:
`34L` was used as the sequence number in `TestManifestListVersions#SEQ_NUM`.
https://github.com/apache/iceberg/blob/085b556cca202db900755b7023e6d6aea0ae24fa/core/src/test/java/org/apache/iceberg/TestManifestListVersions.java#L49

Thus decided to use the same value for the newly added tests as well.

Reply:
Thanks Sumeet!